srcjs/input_binding_fileinput.js

```diff
@@ -181,6 +181,31 @@ $.extend(FileUploader.prototype, FileProcessor.prototype);
 }).call(FileUploader.prototype);
 
 
+function uploadDroppedFilesIE10Plus(el, files) {
```

Comment:
This function looks like it largely duplicates code from `uploadFiles`. Could you refactor the functions so that the code is shared?